### ISSUE TITLE
The word "DotNetNuke" exists in several language .resx files

### ISSUE THREAD

**Issue by EPTamminga** [dnnsoftware/Dnn.Platform#2236]
## Description 
DotNetNuke is the old name of DNNPlatform. Several .resx files still contain the word DotNetNuke i.s.o. DNNPlatform.

## Steps to reproduce
Text search in the unzipped install pack on .resx files containing the word DotNetNuke

## Current result
A list of files containing that word

## Expected result
No files found

## Affected version

* [x] 9.2.1
* [x] 9.2.0
* [x] 9.1.1
* [x] 9.1
* [x] 9.0

## Affected browser

* [x] Chrome
* [x] Firefox
* [x] Safari
* [x] Internet Explorer
* [x] Edge

**Comment by sleupold** [comment]
Do you have a list of files?
Are you referring to resx files included with DNN or separate language packs?

**Comment by EPTamminga** [comment]
@sleupold The basic .resx files in the install .zip:

> GlobalResources.resx
> Authentication.ascx.resx
> Settings.ascx.resx
> Installwizard.aspx.resx
> Installwizard.aspx.de-DE.resx
> Installwizard.aspx.es-ES.resx
> Installwizard.aspx.fr-FR.resx
> Installwizard.aspx.it-IT.resx
> Installwizard.aspx.nl-NL.resx
> UpgradeWizard.aspx.resx
> UpgradeWizard.aspx.de-DE.resx
> UpgradeWizard.aspx.es-ES.resx
> UpgradeWizard.aspx.it-IT.resx
> UpgradeWizard.aspx.nl-NL.resx
> Default Website.template.en-US.resx

**Comment by sleupold** [comment]
I just checked an install of DNN 9.2.0 and you are correct.
I will take care of the German files, Do you plan to submit a PR for the English files or should I take care of it?

**Comment by mitchelsellers** [comment]
There are cases where the copyright notice for the older name should still be included in the texts.  DO we know what/where this usage exists?

**Comment by EPTamminga** [comment]
@sleupold I am willing to generate PR's for all occurrences. The 'DotNetNuke' word in the .resx files is almost always related to the introductory text and not related to copyright notices.

If there is a legal notice that requires DotNetNuke as a word, I will leave the text as is.

@mitchelsellers Is the word **DNNPlatform** the right word to replace the word **DotNetNuke**?

**Comment by sleupold** [comment]
@EPTamminga please do not modify the resource files in other languages, this should be done by the translators - I'll make sure, the installwizard and upgradewizard resx files get copied over, once they are provided (there is a risk, they will be overwritten by the translators otherwise)

**Comment by mitchelsellers** [comment]
@EPTamminga it would depend on the context.  DNN is the actual registered trademark, but it is often referred to as DNNPlatform.  (Such as this repository)

**Comment by EPTamminga** [comment]
@sleupold OK, good to notice. 
I will only create a PR for the English versions.

**Comment by sleupold** [comment]
@EPTamminga 
thanks a lot!

**Comment by sleupold** [comment]
regarding the trademark: in the translation, I usually call it "DNN" and avoid using the name too often. Be aware it will be used for Platform, Evoq and maybe other distributions as well.

**Comment by EPTamminga** [comment]
Pull Request #2240 submitted

**Comment by valadas** [comment]
I would be nice to have the actual core language packs for the provided languages in Github, I know there are teams assigned to localization but when a user of a specific language spots a problem, he could submit a pull request and those officially assigned to that team could merge it in. I remember seeing many references to host and admin menus in some place of the french language pack, those references are no longer valid with the persona bar. Those things are very time consuming to work on independently but as users of different languages spot these during usage they could submit a quick pull request for a small correction.

**Comment by EPTamminga** [comment]
We (XCESS) are the official translators for Dutch.
I tried this before (https://github.com/XCESS-NL/DNN.nl-NL). The repoaitory includes a gitignore that enables to have a normal full DNN installation locally and only sync Dutch resx files if you have a correction. I did some promotions in various channels, without success. 
The official translators have (Windows) tools made by @donker to make differential translations more efficient (changed, new, deleted text, statistics).
If we want community contributions for translation (I favor that), we must do more than put a language pack in a repo.
